frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/SetupTab/SetupTabContent/SwitchRow/SwitchRow.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+import classes from './SwitchRow.module.css';
+import { Paragraph, Switch } from '@digdir/design-system-react';
+
+export type SwitchRowProps = {
+  checked: boolean;
+  onSave: (checked: boolean) => void;
+  label: string;
+};
+
+/**
+ * @component
+ *    Displays a Switch component with a label to the right
+ *
+ * @property {boolean}[chekded] - If the switch is on or off
+ * @property {function}[onSave] - Function to be executed when clicking the switch
+ * @property {boolean}[label] - The label of the switch
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const SwitchRow = ({ checked, onSave, label }: SwitchRowProps): ReactNode => {
+  return (
+    <Switch
+      size='small'
+      className={classes.switch}
+      checked={checked}
+      onChange={(e) => onSave(e.target.checked)}
+    >
+      <Paragraph size='small'>{label}</Paragraph>
+    </Switch>
+  );
```

Comment:
Hvorfor heter denne SwitchRow? Basert på navnet så forventet jeg at dette var en rekke med "switch'er" i en rad. Når jeg leser implementasjonen, så ser jeg at dette er en wrapper komponent rundt switch, men den gjør ikke noe annet en å re-implementere switch relativt as is? Kunne vi droppet denne wrapper komponenten og bare brukt switch direkte? 😊

Reply:
Fikser 😄